python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -297,6 +254,95 @@ cdef object _process_metadata(object df,
     return df
 
 
+cdef class CudfChunkedParquetReader(ChunkedParquetReader):
+    """
+    Subclass of the pylibcudf chunked parquet reader
```

Comment:
Was trying to be too clever here.

Since we always read all chunks from the chunkedparquetreader, there's no point keeping it a class, so I've turned this into a function in my next commit